.github/workflows/main.yml

```diff
@@ -16,15 +16,15 @@ jobs:
     name : TestIOS 
     # runs-on: will be set to macos-latest when running on actual GHA. 
     # *** runs-on: ubuntu-latest is used when running via act on mac os. ***
-    runs-on: macos-13 
```

Comment:
Are these updates on the runner because of this feature?

Reply:
Github started failing watchOS and tvOS with the Xcode version it was set to not being present. So I updated it but I know there have been other pr's where that wasn't an issue. Maybe it was fixed on the GitHub side, I can remove it if we want to verify.